### PR TITLE
Content changes and pics

### PR DESCRIPTION
**Pull request by kiddi**

### CODE REVIEW

File: src/routes/team/Team.js [hunk 1]
```diff
@@ -15,31 +15,31 @@ class Team extends Component {
...
+              alt="Vignir Örn Guðmundsson"
             />
             <h2>Vignir Örn Guðmundsson</h2>
             <h2>SUT</h2>
           </div>
           <div className={s.person}>
             <img
-              src="/img/Team/kristjanmik.jpg"
-              alt="Kristján Ingi Mikaelsson"
+              src="/img/Team/gudbjorg.jpg"
```
Comment (johannth) [inline src/routes/team/Team.js]: These links will be broken because of the capital T, see #34

**Review by johannth — CHANGES_REQUESTED** [review]

**Comment by johannth** [comment]
Lol, I really didn't really check the urls on #34, just replaced T with t. Maybe merge that first and then rebase this one off master?

File: src/routes/team/Team.js (hunk 1 above)
Comment (kiddi) [reply src/routes/team/Team.js]: Will fix. Strange that it works locally though.

**Comment by kiddi** [comment]
just changed here :)

**Comment by kiddi** [comment]
Just changed them on this branch.

**Review by johannth — APPROVED** [review]

**Comment by johannth** [comment]
CI failed again on master. Don't have time to investigate, will deploy locally. Probably some env issue with CodeShip.

**Pull request merged**